src/main/java/uk/gov/hmcts/reform/prl/services/UpdatePartyDetailsService.java

```diff
@@ -107,10 +117,91 @@ public Map<String, Object> updateApplicantRespondentAndChildData(CallbackRequest
                 setApplicantOrganisationPolicyIfOrgEmpty(updatedCaseData, ElementUtils.unwrapElements(applicantList.get()).get(0));
             }
         }
-
+        cleanUpCaseDataBasedOnYesNoSelection(updatedCaseData, caseData);
+        log.info("CaseData Respondents " + caseData.getRespondents());
+        log.info("CaseData RespondentsFL401 " + caseData.getRespondentsFL401());
+        log.info("Map Respondents " + updatedCaseData.get("respondents"));
+        log.info("Map RespondentsFL401 " + updatedCaseData.get("respondentsFL401"));
         return updatedCaseData;
```

Comment:
please remove all the logger statements.

Reply:
Removed